packages/text-sets/src/cli.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable no-constant-condition, no-await-in-loop, no-console */
```

Comment:
That's a lot of things to disable 😄 

Instead of `eslint-comments/disable-enable-pair`, just put an `eslint-enable` comment at the bottom of the file.

Instead of `no-constant-condition`, replace `while (true)` with `while (lastTextSetId === null)`?

Reply:
Did the eslint part, and changed the loop a little (`lastTextSetId === null` wouldn't work), so right now the condition is redundant just to make the eslint happy (we have `break` in the loop that skips generating the last image twice).